airbyte-oauth/build.gradle

```diff
@@ -12,6 +12,7 @@ dependencies {
     implementation libs.appender.log4j2
     implementation libs.aws.java.sdk.s3
     implementation libs.aws.java.sdk.sts
+    implementation libs.sts
```

Comment:
I believe we don't need this since the v2 S3 module is not used in the package

Reply:
Good catch, removing!